packages/react/src/badge/Badge.tsx

```diff
@@ -0,0 +1,44 @@
+import { type VariantProps, cva } from 'cva';
+import { type Ref, forwardRef } from 'react';
+
+type BadgeProps = VariantProps<typeof badgeVariants> & {
+  children?: React.ReactNode;
+  className?: string;
+};
+
+const badgeVariants = cva({
+  base: ['inline-flex w-fit items-center justify-center gap-1.5 rounded-lg'],
```

Comment:
Slenge på en liten fail-safe her slik at ikonet ikke krymper ved lange tekster?

```suggestion
  base: ['inline-flex w-fit items-center justify-center gap-1.5 rounded-lg  [&_svg]:shrink-0'],
```